### ISSUE TITLE
Request: Upload gradle-velocity-plugin to the gradle plugin portal

### ISSUE THREAD

**Issue by johan-onCreate** [shevek/gradle-velocity-plugin#1]
It would be useful if this plugin could be uploaded to the gradle plugin portal.
By that you could use the plugins-block instead of buildscript -> dependencies.

https://plugins.gradle.org

**Comment by shevek** [comment]
I've tried to do that several times over the evolution of the gradle plugins repository, and I've found it to be an INCREDIBLE pain in the arse, which was made only more ironic by the rampant trumpeting of how much easier it is than using Central. It looks as if they've now dropped the requirement to use bintray (thank goodness) so I might have another go at this.

Since I publish everything using the stdproject plugin, the easiest way would be to integrate the current gradle plugin publishing method into stdproject, at which point all my plugin projects will become available on the portal.

**Comment by shevek** [comment]
Yes, on first pass, this looks a lot better, thank you for poking me.
